python/lsst/sattle/sattlePy.py

```diff
@@ -2,59 +2,75 @@
 import numpy as np
 from astropy.time import Time
 
-import lsst.pex.config as pexConfig
-import lsst.pipe.base as pipeBase
 import lsst.sphgeom as sphgeom
 import warnings
+from typing import Any, Optional
 
 
 __all__ = ["SattleConfig", "SattleTask"]
```

Comment:
Please document throughout following doc conventions, e.g. https://developer.lsst.io/python/numpydoc.html#py-docstring-class-structure for classes